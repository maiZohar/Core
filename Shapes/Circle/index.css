
.circle_component {
    display: inline-block;
    border: 1px solid #000;
    background-color: #fff;
    border-radius: 50%;
}
