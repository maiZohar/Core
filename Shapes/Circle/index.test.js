import React from 'react';
import TestRenderer from 'react-test-renderer';
import Circle from './index';

describe('Circle', () => {
    it('should render default state', () => {
        const component = TestRenderer.create(<Circle />).toJSON();
        expect(component).toMatchSnapshot();
    });

    it('should render state with props', () => {
        const component = TestRenderer.create(
            <Circle
                color="color_mock"
                borderColor="border_color_mock"
                size={42}
                dataAutomations="data_automations_mock"
                className="class_name_mock"
            />
        ).toJSON();
        expect(component).toMatchSnapshot();
    });
});
