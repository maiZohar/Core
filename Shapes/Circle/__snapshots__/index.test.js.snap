// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Circle should render default state 1`] = `
<div
  className="circle_component"
  data-automations="circle_component"
  onClick={[Function]}
  style={
    Object {
      "backgroundColor": undefined,
      "borderColor": undefined,
      "height": "14px",
      "minWidth": "14px",
    }
  }
/>
`;

exports[`Circle should render state with props 1`] = `
<div
  className="circle_component class_name_mock"
  data-automations="data_automations_mock"
  onClick={[Function]}
  style={
    Object {
      "backgroundColor": "color_mock",
      "borderColor": "border_color_mock",
      "height": "42px",
      "minWidth": "42px",
    }
  }
/>
`;
